api/src/main/java/io/druid/data/input/impl/FloatDimensionSchema.java

```diff
@@ -27,10 +27,16 @@
 {
   @JsonCreator
   public FloatDimensionSchema(
-      @JsonProperty("name") String name
+      @JsonProperty("name") String name,
+      @JsonProperty("multiValueHandling") MultiValueHandling multiValueHandling
```

Comment:
ah, I see. Yeah, even so, IMO better to defer until it's actually used by something.